charts/router/templates/router-clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if .Capabilities.APIVersions.Has (include "rbacApiVersion" .) }}
+kind: ClusterRole
+apiVersion: {{ template "rbacApiVersion" . }}
+metadata:
+  name: deis:deis-router
```

Comment:
@krancour of course it is. More than that: it's recommended for "global" (not namespaced) resources.
Look at clusterroles, defined by kubernetes itself:

    $ kubectl get clusterrole
    NAME                                           AGE
    ...
    system:auth-delegator                          1h
    system:basic-user                              6h
    system:controller:attachdetach-controller      1h
    system:controller:certificate-controller       1h
    system:controller:cronjob-controller           1h
    system:controller:daemon-set-controller        1h
    system:controller:deployment-controller        1h
    system:controller:disruption-controller        1h
    system:controller:endpoint-controller          1h
    system:controller:generic-garbage-collector    1h
    system:controller:horizontal-pod-autoscaler    1h
    system:controller:job-controller               1h